### DISCUSSION TITLE
创建网站Java运行环境的时候报错

### DISCUSSION THREAD

**Discussion by liuzedong-hub** [1Panel-dev/1Panel#6180] [Q&A]
Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit [1mhttps://bitnami.com/enterprise[0m
[2Kjava_run_space  | [38;5;6m [38;5;5m01:59:11.52 [0m[38;5;2mINFO [0m ==>

**Comment by wanghe-fit2cloud** [comment]
看错误信息，可能是 Tanzu Application Catalog 的免费或试用版本吧，不是很清楚，和 1Panel 应该没有关系。

**Reply by liuzedong-hub** [reply]
那就不能使用1panel创建Java运行环境了吗
